src/App/components/InputGroup/index.js

```diff
@@ -118,7 +117,7 @@ const InputGroup = ({
                             <input {...attrs} />}{"\n"}
                 </>
             }
-            {errorMessage && <><div className="help-block">{errorMessage}</div>{"\n"}</>}
+            {errorMessage && <><div className="help-block"><i className="material-icons">error</i>{errorMessage}</div>{"\n"}</>}
```

Comment:
The icon should be inside the field, and not before the error message. Like this:
![image](https://user-images.githubusercontent.com/44967773/174800658-430bcf22-e351-48b3-ae35-9f765cc3e23c.png)

This also removed the error icon from input field
![image](https://user-images.githubusercontent.com/44967773/174800731-90561425-7790-42b8-a770-a56dfd5c1e4d.png)